cmd/entrypoint/entrypoint.go

```diff
@@ -188,9 +189,14 @@ func Main(ctx context.Context, Version string, args ...string) error {
 
 	if !demoMode {
 		group.Go("watcher", func(ctx context.Context) error {
+			intv, err := strconv.Atoi(env("AMBASSADOR_WATCH_INTERVAL_SEC", "0"))
```

Comment:
The original reason was to keep the config closer together instead of spread out. But to the point of having a general mechanism for config or a config object of sorts, while I prefer to have one, I'm not super opposed to moving it closer to it actual usage in this instance.